Lifecycle app/lcHomePage.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
+	var  mn = $(".main-nav");
+    mns = "main-nav-scrolled";
+    hdr = $('header').height();
+
+$(window).scroll(function() {
+  if( $(this).scrollTop() > hdr ) {
+    mn.addClass(mns);
+    mn.removeClass('.main-nav');
+  } else {
+    mn.removeClass(mns);
+  }
+});
```

Comment:
Small whitespace issue, `$(window).scroll` and it's function all should be have two more leading spaces, being that they are within `$(document).ready`.